.github/workflows/test.yaml

```diff
@@ -63,4 +66,20 @@ jobs:
 
       - run: patchright install --no-shell chromium
 
-      - run: pytest tests/${{ matrix.test }}.py
+      - name: Install Playwright and Chromium
+        run: |
+          uv pip install playwright
+          uv run playwright install chromium
+          uv run patchright install chrome
+
+      - name: Launch Chrome with remote debugging
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y google-chrome-stable
```

Comment:
we should use an off-the-shelf action for this instead of manually apt-installing, otherwise it will add a lot of time to every test